variables.tf

```diff
@@ -165,6 +165,12 @@ variable "security_groups" {
   default     = []
 }
 
+variable "root_disk_size" {
+  type        = number
+  description = "The size of the OpenVPN instance's root disk in GiB."
+  default     = 8
```

Comment:
Do we care about ensuring a minimum size?

Reply:
Are you talking about adding a check so that no one can make this value smaller than <X>?

Reply:
Yes that is what I was referring to just as a "cover the bases" thing.

Reply:
As is usual with EC2 instances launched from an AMI, the minimum size for the root disk is the size of the AMI. This size is [currently 8GB](https://github.com/cisagov/openvpn-packer/blob/0bd9c0a84ba97bbbfef88129690a10f5f52fe8e2/src/packer.pkr.hcl#L100).  If you specify a smaller size then the instance will fail to launch.

I agree with @dav3r that such a check isn't necessary here.